librz/bin/format/ne/ne.c

```diff
@@ -118,6 +118,46 @@ static void ne_sanitize_name(char *name, ut16 count) {
 	}
 }
 
+static const char *get_reloc_type_name(const ut8 src_type, const ut8 flag) {
+#define CONCAT(a, b) a b
```

Comment:
Rename `CONCAT` to something else like `CONCAT_RELOC_STR`, because some headers might define this.